bin/main.ml

```diff
@@ -30,7 +31,9 @@ let set_common c =
   Clflags.debug_dep_path := c.debug_dep_path;
   Clflags.debug_findlib := c.debug_findlib;
   Clflags.dev_mode := c.dev_mode;
-  Printf.eprintf "Workspace root: %s\n" c.root;
+  Clflags.verbose := c.verbose;
+  if !Clflags.verbose then
+    Printf.eprintf "Workspace root: %s\n" c.root;
```

Comment:
Done.

I did this by moving the logging to Future.Scheduler.go, where a log is available. (That made more sense to me than making a log available in set_common, because logging functions might want to use `Clflags`. I can change this if you prefer it the other way around).

Also, when `Clflags.verbose` is set, `Log.info` is now copied to stderr, so that these messages still appear on the screen with `--verbose`.